utils/page_objects/base_collections/header_navigation.js

```diff
@@ -0,0 +1,11 @@
+class HeaderNavigationMenu {
```

Comment:
There is no need to create a separate collection for a header in your framework. You can add it as a separate Page Object or add `menuLoginButton` as a single element of your `BasePage` class.